packages/nimble-components/package.json

```diff
@@ -16,6 +16,7 @@
     "build-storybook": "build-storybook -o dist/storybook",
     "build-components": "tsc -p ./tsconfig.json",
     "build-components:watch": "tsc -p ./tsconfig.json -w",
+    "build-components:version": "tsc --version",
```

Comment:
I'm not opposed to logging the tsc version if it's helpful, but I'd vote to give the command a name like `tsc-version` or `print-tool-versions`. Keeping `build-components` in the name makes me think that it will actually build something.